scripts/local-deploy-bootstrap.py

```diff
@@ -374,14 +374,26 @@ def get_from_pypi(app, version, pypi='https://pypi.python.org/simple/'):
     return expected_name
 
 
-def build_virtualenv(bootstrap=False):
+def infer_compat_version():
+    """Return appropriate compat level for the python version"""
+    if sys.version_info.major == 2:
+        return 4
+    elif sys.version_info.major == 3:
+        return 5
+
+
+def build_virtualenv(bootstrap=False, compat=None):
     """Build a virtualenv for CWD, bootstrapping if necessary"""
     if not binary_available('build-virtualenv') and not bootstrap:
         raise Exception("build-virtualenv can't be found on PATH")
 
     # Already bootstrapped
-    if call(('build-virtualenv',)) == 0:
-        return
+    if compat:
+        if call(('build-virtualenv', '--compat={}'.format(compat))) == 0:
+            return
+    else:
+        if call('build-virtualenv') == 0:
+            return
```

Comment:
I'd suggest building the arguments in the branch, rather than repeating all the logic. Not that there is *that* much of it...